partner_search_alias/views/res_partner_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_partner_tree_alias">
+        <field name="name">res.partner.tree</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_tree" />
+        <field name="arch" type="xml">
+            <field name="translated_display_name" position="after">
+                <field name="search_alias" />
+            </field>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_partner_form_alias">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <field name="vat" position="after">
```

Comment:
Let's move the field somewhere less obtrusive.
```suggestion
            <field name="ref" position="after">
```

Reply:
I updated the code.